packages/core/addon/models/fragments/presentation.js

```diff
@@ -5,9 +5,10 @@
 
 import DS from 'ember-data';
 import Fragment from 'ember-data-model-fragments/fragment';
+import { Copyable } from 'ember-copy';
 import { fragmentArray } from 'ember-data-model-fragments/attributes';
 
-export default Fragment.extend({
+export default Fragment.extend(Copyable, {
```

Comment:
Is this necessary? The Fragment should already have that mixin
https://github.com/lytics/ember-data-model-fragments/blob/master/addon/fragment.js#L70

Reply:
yeah it's necessary, blew up without it in 2.x app...